inst/odin/lancelot.R

```diff
@@ -20,16 +20,27 @@ update(time) <- (step + 1) * dt
 ## output number of individuals vaccinated by age and vaccine stage
 ## For example, for E, we sum over n_E_next_vacc_class (those moving vaccine
 ## stage without progressing disease stages)
-## vaccinated S
-n_S_vaccinated[, ] <- n_S_next_vacc_class[i, j]
+## Note that individuals making vaccine skip moves are counted in all classes
+## they skip over
+n_S_vaccinated[, ] <- n_S_next_vacc_class[i, j] +
+  (if (j >= vacc_skip_from && j < vacc_skip_to) n_S_vacc_skip[i]
+   else 0)
```

Comment:
the code is ok but maybe the comment could be a bit clearer. e.g. for those who skip a class, in the outputs, we count them as if they had gone through all the successive classes (possibly without staying any time in the intermediate classes) - I think that would be a bit clearer

Reply:
Hopefully a bit clearer now